### PR TITLE
chore: Add changelog entries in prep for release 1.7.0

### PR DESCRIPTION
**Pull request by mjameswh**
## What changed

- Updated CHANGELOG.md file to include 1.7.0 release

### CODE REVIEW

**Review by bergundy — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,75 @@ All notable changes to this project will be documented in this file.
...
+    histories
+  );
+  for await (const result of replayResults) {
+    const { workflowId, runId, error } = result;
+    // error is either undefined, a ReplayError or a DeterminismViolationError
+  }
+  ```
+
+- :boom: [`worker`] `WorkerOptions.shutdownGraceTime` no longer forcefully shuts the worker down.
+  Set `WorkerOptions.shutdownForceTime` to force shutdown. ([#1072](https://github.com/temporalio/sdk-typescript/pull/1072))
```
Comment (lorensr) [inline CHANGELOG.md:32]:
```suggestion
- :boom: [`worker`] `WorkerOptions.shutdownGraceTime` no longer forcefully shuts the worker down. Now, when `shutdownGraceTime` passes, the worker just sends Cancellation to running Activities.
  Set `WorkerOptions.shutdownForceTime` to force shutdown. ([#1072](https://github.com/temporalio/sdk-typescript/pull/1072))
```

accurate?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,75 @@ All notable changes to this project will be documented in this file.
...
+  const replayResults = Worker.runReplayHistories(
+    {
+      workflowsPath: require.resolve('./workflows'),
+      // ...
+    },
+    histories
+  );
+  for await (const result of replayResults) {
+    const { workflowId, runId, error } = result;
+    // error is either undefined, a ReplayError or a DeterminismViolationError
```
Comment (lorensr) [inline CHANGELOG.md:27]:
```suggestion
    // error is either undefined, a ReplayError, or a DeterminismViolationError
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,75 @@ All notable changes to this project will be documented in this file.
...
+  }
+  ```
+
+- :boom: [`worker`] `WorkerOptions.shutdownGraceTime` no longer forcefully shuts the worker down.
+  Set `WorkerOptions.shutdownForceTime` to force shutdown. ([#1072](https://github.com/temporalio/sdk-typescript/pull/1072))
+
+- :boom: [`testing`] Use temporal CLI to power local test environment ([#1077](https://github.com/temporalio/sdk-typescript/pull/1077))
```
Comment (lorensr) [inline CHANGELOG.md:34]:
```suggestion
- :boom: [`testing`] Use [Temporal CLI](https://github.com/temporalio/cli/#temporal-cli) to power local test environment ([#1077](https://github.com/temporalio/sdk-typescript/pull/1077))
```

**Review by lorensr — APPROVED** [review]

**Comment by mjameswh** [comment]
@lorensr Thanks for suggestions.

**Pull request merged**